### PR TITLE
WIP: Separate re-init per node

### PR DESCRIPTION
**Pull request by mrogers950**

### CODE REVIEW

**Comment by mrogers950** [comment]
@JAORMX @jhrozek My manual tests with this approach did not work, so I still need to debug. Please take a quick look to see what I might be missing.

File: pkg/common/util.go [hunk 1]
```diff
@@ -127,22 +127,22 @@ func GetReinitDaemonSetName(name string) string {
 
 // RestartFileIntegrityDs restarts all pods that belong to a given DaemonSet. This can be
 // used to e.g. remount a configMap after it had changed or restart a FI DS after a re-init
-// had happened
-func RestartFileIntegrityDs(c client.Client, dsName string) error {
+// had happened. Node is optional and will only restart the pods on said node if specified..
+func RestartFileIntegrityDs(c client.Client, dsName, node string) error {
```
Comment (JAORMX) [inline pkg/common/util.go:131]: Does this apply to the re-init daemonset too? thought it only applied to the regular daemonset

File: pkg/controller/fileintegrity/fileintegrity_controller.go [hunk 2]
```diff
@@ -442,11 +450,87 @@ func aidePauseScript() *corev1.ConfigMap {
...
+						MountPath: "/hostroot",
+					},
+					{
+						Name:      common.AideReinitScriptConfigMapName,
+						MountPath: "/scripts",
+					},
+				},
+			},
+		},
+		// make this an endless loop
```
Comment (JAORMX) [inline pkg/controller/fileintegrity/fileintegrity_controller.go:482]: I think we need to add the fileintegrity tolerations here too.

**Comment by mrogers950** [comment]
Closing, going to approach differently post-rewrite

**Pull request closed without merging**